.cargo-husky/hooks/pre-commit

```diff
@@ -2,9 +2,6 @@
 
 set -e
 
-# make clippy check all rust files again (bug in clippy)
-find . -name '*.rs' | xargs touch
```

Comment:
Awesome! This was driving me nuts because it made Cargo recompile Monster every time I did a commit locally.